skuba-update/test/os/suse/Dockerfile.builder

```diff
@@ -1,4 +1,4 @@
 FROM registry.opensuse.org/opensuse/tumbleweed
 
-RUN zypper ref && zypper -n in rpm-build rpmdevtools createrepo
-RUN rm /var/run/reboot-needed
\ No newline at end of file
+RUN zypper ref && zypper -n in rpm-build rpmdevtools createrepo libcreaterepo_c-devel
```

Comment:
Current CI checks the file `/var/run/reboot-needed` is existed.

Install `createrepo_c`, it won't trigger the reboot-needed file. I manually found a package `libcreaterepo_c-devel` that would trigger the reboot-needed file.

Reply:
I think it would be better to create a dummy RPM package **we control** that always triggers the reboot-needed file.

By doing that we would have full control over the outcome of the test and everything would be more clear.